doc/classes/Engine.xml

```diff
@@ -110,15 +110,16 @@
 			<return type="Dictionary" />
 			<description>
 				Returns the current engine version information in a Dictionary.
-				[code]major[/code]    - Holds the major version number as an int
-				[code]minor[/code]    - Holds the minor version number as an int
-				[code]patch[/code]    - Holds the patch version number as an int
-				[code]hex[/code]      - Holds the full version number encoded as a hexadecimal int with one byte (2 places) per number (see example below)
-				[code]status[/code]   - Holds the status (e.g. "beta", "rc1", "rc2", ... "stable") as a String
-				[code]build[/code]    - Holds the build name (e.g. "custom_build") as a String
-				[code]hash[/code]     - Holds the full Git commit hash as a String
-				[code]year[/code]     - Holds the year the version was released in as an int
-				[code]string[/code]   - [code]major[/code] + [code]minor[/code] + [code]patch[/code] + [code]status[/code] + [code]build[/code] in a single String
+				[code]major[/code]     - Holds the major version number as an int
+				[code]minor[/code]     - Holds the minor version number as an int
+				[code]patch[/code]     - Holds the patch version number as an int
+				[code]hex[/code]       - Holds the full version number encoded as a hexadecimal int with one byte (2 places) per number (see example below)
+				[code]status[/code]    - Holds the status (e.g. "beta", "rc1", "rc2", ... "stable") as a String
+				[code]build[/code]     - Holds the build name (e.g. "custom_build") as a String
+				[code]hash[/code]      - Holds the full Git commit hash as a String, or "unknown" if unavailable
+				[code]timestamp[/code] - Holds the Git commit date UNIX timestamp in seconds as an int, or 0 if unavailable
+				[code]year[/code]      - Holds the year the version was released in as an int
```

Comment:
Is `year` still really necessary?

Reply:
Unlike `timestamp`, `year` is still available if Git commit data is not available. So I'd prefer to keep it in.